django/researchdata/models.py

```diff
@@ -69,5 +70,23 @@ class Answer(models.Model):
     def __str__(self):
         return str(self.answer_text)[0:40]
 
+    def save(self, *args, **kwargs):
+        """
+        When a new model is saved email the research team
+        """
+
+        # Check if this is a new answer
+        if(self.meta_created_datetime is None):
+            # Send email alert to research team
+            send_mail(
+                        'CLiC Calendar: New Answer',
```

Comment:
The indentation looks wrong here - possible tabs/spaces issue? I'd suggest putting the first arg straight after the `(` rather than a new line. I only mention this since I've put other comments for the same bit of code!